File: BEE 1041.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        float x = input.nextFloat();
        float y = input.nextFloat();
        if (x == 0 && y == 0){
            System.out.printf("Origem\n");
        }
        else if (x != 0 && y == 0){
            System.out.printf("Eixo X\n");
        }
        else if (x == 0 && y != 0){
            System.out.printf("Eixo Y\n");
        }
        else if (x > 0 && y > 0){
            System.out.printf("Q1\n");
        }
        else if (x < 0 && y > 0){
            System.out.printf("Q2\n");
        }
        else if (x < 0 && y < 0){
            System.out.printf("Q3\n");
        }
        else{
            System.out.printf("Q4\n");
        }
    }
}

File: BEE 1041.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var x = parseFloat(line.shift());
var y = parseFloat(line.shift());

if (x == 0 && y == 0){
    console.log("Origem");
}
else if (x != 0 && y == 0){
    console.log("Eixo X");
}
else if (x == 0 && y != 0){
    console.log("Eixo Y");
}
else if (x > 0 && y > 0){
    console.log("Q1");
}
else if (x < 0 && y > 0){
    console.log("Q2");
}
else if (x < 0 && y < 0){
    console.log("Q3");
}
else{
    console.log("Q4");
}